app/views/ems_container/_main.html.haml

```diff
@@ -12,7 +12,5 @@
   .col-sm-12.col-md-12.col-lg-6
     = render :partial => "shared/summary/textual", :locals => {:title => _("Relationships"),
                                                                :items => textual_group_relationships}
-    = render :partial => "shared/summary/textual", :locals => {:title => _("Overview"),
-                                                               :items => textual_group_topology}
```

Comment:
> @zeari Probably wanted to skip this one too? ;)

That was on purpose! Do we want to keep that link on the summary page?
![screencapture-localhost-3000-ems_container-5-1468835545162](https://cloud.githubusercontent.com/assets/11256940/16911365/be7bf820-4ce6-11e6-8652-f6128c9c6841.png)